ddrp/Region region
string objective_name
int8 state
